src/App.jsx

```diff
@@ -1,20 +1,13 @@
 import { Layout, Menu } from 'antd'
-const { Header, Footer, Sider, Content } = Layout
+import { StyledSider, StyledHeader, StyledContent, StyledFooter, StyledImg } from './atoms.js'
 import './App.css'
 
 function App() {
   return (
   <Layout>
-    <Header className='header'>Khinkalnya</Header>
+    <StyledHeader className='header'>Khinkalnya</StyledHeader>
     <Layout>
-      <Sider
-        className='sider'
-        style={{
-          height: 'calc(100% - 64px)',  // viewport height minus header height
-          position: 'sticky',
-          top: '64px',
-        }}
-      >
+      <StyledSider className='sider'>
```

Comment:
We don't have style `.sider`
```suggestion
      <StyledSider>
```

Reply:
Fixed